packages/unchained-client/src/evm/ethereum/parser/weth.ts

```diff
@@ -37,9 +41,9 @@ export class Parser implements SubParser<Tx> {
     this.wethContract = (() => {
       switch (args.chainId) {
         case 'eip155:1':
-          return WETH_CONTRACT_MAINNET
+          return WETH_TOKEN_CONTRACT_ADDRESS
         case 'eip155:3':
```

Comment:
We don't support testnets, can we remove case/testnet references here while at it?

Reply:
can do!

Reply:
hasta la vista baby https://github.com/shapeshift/web/pull/7687/commits/5a9bebd986defb200d7ae78d3f4fb6e87103f91e